### ISSUE TITLE
Missing support for block images

### ISSUE THREAD

**Issue by spaal** [asciidoctor/atom-asciidoc-preview#267]
## Description

Images can be positioned as inline or block image. While inline images are correctly shown in the asciidoc preview window, block images are not. When I publish the asciidoc document to PDF, e.g. by using asciidoctor, the image appears in the PDF document as block image.

- Atom version: 1.11.2
- OS: Linux Mint 18.3
- `asciidoc-preview` version: 2.9.2
- `language-asciidoc` version: 1.6.2

## Syntax example

Inline images defined as below are shown in the asciidoc-preview window:
```adoc
image:MyImage.png[]
```

Block images defined as below are NOT shown in the asciidoc-preview window but the statement is shown:
```adoc
image::MyImage.png[]
```

## References
https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#images
https://github.com/asciidoctor/docker-asciidoctor

**Comment by ldez** [comment]
Hello, I don't reproduce your issue.

Block images and inline images are both supported.

**Comment by ldez** [comment]
You can encode the path of your image like that:

```asciidoc
image::My%20Image.png[alt]
```

**Comment by spaal** [comment]
Yeah but then the PDF generation with asciidoctor fails with image not found error as it probably does not substitute %20 with a space. The workaround using underscores instead of spaces works for atom-asciidoc-preview and asciidoctor.

**Comment by mojavelinux** [comment]
This will be resolved once the upgrade to 1.5.6 is complete as spaces are now permitted by the core processor.
